packages/test/v0.7/permissionValidator.test.ts

```diff
@@ -491,6 +492,45 @@ describe("Permission kernel Account", () => {
         TEST_TIMEOUT
     )
 
+    test(
+        "Smart account client send transaction with TimestampPolicy",
+        async () => {
+            const timestampPolicy = await toTimestampPolicy({})
```

Comment:
In this case I guess there's no time limit set since the defaults are `0`.
Haven't looked at contract code but assuming it works same as validation time stamps params as before.
So the tx isn't timebound. Just for a sanity check would be good to have a testcase with limit set and see if it works or not.